source/d_qgis_plugin.rst

```diff
@@ -118,162 +141,136 @@ On load of the model the following files are retrieved from the server:
 
-**Post-processing in Lizard**
-Works only for users with this module. Enables storing results in the cloud, automated postprocessing of waterdepth and water levels maps, time of arrival, flood hazard rating and damage estimations (only available in the Netherlands at the moment). Contact us at servicedesk@nelen-schuurmans.nl if you like to use this option and don't have access yet.
+**Post-processing in Lizard** 
+This is a feature that is only availble for users of organisations that have a Lizard account. It enables you to store the results in the cloud and it triggers some automated post-processing. It will generate various maps of water depth, water levels, time of arrival, flood hazard rating and damage estimations. The damage estimations are only available in the Netherlands. Contact us at servicedesk@nelen-schuurmans.nl if you like to use this option and don't have access yet.
 	
-
-When starting a new simulation, you need to define a name for the simulation. Other users within your organisation will see this. It can also be used to look up simulations later. 
+The next step is to name the simulation. You and other users within your organisation will be able to find this simulation and its results based on the name. It can also be used to look up simulations later. 
 
 .. figure:: image/d_qgisplugin_apiclient_new_simulation.png
     :alt: Choose new simulation 
 
-
-The first step in any simulation is choosing the simulation duration:
-
+Afterwhich the simulation duration can be defined:
 
 .. figure:: image/d_qgisplugin_apiclient_choose_duration.png
     :alt: Choose duration
 	
-The next steps depend on the selection from the initial screen of the wizard. If not checked, these steps will be omitted by the wizard.
+Depending on the scenario option you have defined, the wizard will allow you provide the details of your scenario options.
 
 **Initial conditions**
 
-Initial conditions either refer to the use of saved state file, or the use of initial water level in 1D, 2D or groundwater (2D). 
+Initial conditions either refer to the use of a saved state file, or the use of initial water level in 1D, 2D surface or ground water domain. 
 
 .. figure:: image/d_qgisplugin_apiclient_initialconditions_start.png
     :alt: Choose initial conditions
 	
 1D options:
 
 - Predefined: this refers to the initial water level as defined in the column initial_waterlevel in the connection nodes in the spatialite. 
-- Global value: this would be a generic initial waterlevel value in m MSL which is applied in all 1D nodes of the model.
+- Global value: this would be a generic initial waterlevel value in m MSL, which will then be applied in all 1D nodes of the model.
 
-2D options:
+2D Surface Water options:
 
 - Raster: this refers to the initial water level raster as uploaded with the model to the model databank.
```

Comment:
welke je hebt benoemd in je global settings? En kan je hier dus ook nog niet de max en min ave opgeven??